src/common/button.tsx

```diff
@@ -1,9 +1,7 @@
-const Button = ({
-  buttonLabel,
-  className
-}: {
-  buttonLabel: string
-  className: string
-}): JSX.Element => <button className={className}>{buttonLabel}</button>
+import { ButtonProps } from '../types/interfaces'
```

Comment:
leave props interfaces inside of component or in separate file in the same folder